Framework/DataHandling/src/LoadILLPolarizedDiffraction.cpp

```diff
@@ -323,9 +323,15 @@ LoadILLPolarizedDiffraction::initStaticWorkspace(const NXEntry &entry) {
 /**
  * Runs LoadInstrument as child to link the instrument to workspace
  * @param workspace : workspace with data from the first entry
+ * @param startTime :: the date the run started, in ISO compliant format
  */
 void LoadILLPolarizedDiffraction::loadInstrument(
-    API::MatrixWorkspace_sptr workspace) {
+    API::MatrixWorkspace_sptr workspace, std::string startTime) {
```

Comment:
This is confusing, is it start_time or run_start? Make it consistent everywhere.